presto-raptor/src/main/java/com/facebook/presto/raptor/storage/organization/OrganizationSet.java

```diff
@@ -22,13 +22,15 @@
 
 import static com.google.common.base.MoreObjects.toStringHelper;
 import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.toSet;
 
 public class OrganizationSet
 {
     private final long tableId;
     private final boolean tableSupportsDeltaDelete;
     private final Map<UUID, Optional<UUID>> shardsMap;
     private final OptionalInt bucketNumber;
+    private int priority;
 
     public OrganizationSet(long tableId, boolean tableSupportsDeltaDelete, Map<UUID, Optional<UUID>> shardsMap, OptionalInt bucketNumber)
```

Comment:
this should delegate to the more generic one (the one below) rather than the other way.